nexus/src/external_api/params.rs

```diff
@@ -242,20 +242,23 @@ pub struct DiskCreate {
     pub identity: IdentityMetadataCreateParams,
     /// id for snapshot from which the Disk should be created, if any
     pub snapshot_id: Option<Uuid>, // TODO should be a name?
+    /// id for image from which the Disk should be created, if any
+    pub image_id: Option<Uuid>, // TODO should be a name?
     /// size of the Disk
     pub size: ByteCount,
+    /// block size for this disk
+    pub block_size: ByteCount,
```

Comment:
Could we enhance this comment a bit? It acts as api documentation and I think it'd be better to be more explicit. e.g. add any useful context like what the default is (if any) or examples of potential valid options.

Reply:
definitely, done in df508d22